_data/projects/scribe-ios.yml

```diff
@@ -0,0 +1,13 @@
+name: Scribe-iOS
+desc: Keyboards for language learners with translation, verb conjugation and more!
+site: https://github.com/scribe-org/Scribe-iOS
+tags:
+  - swift
+  - ios
+  - language-learning
+  - wikidata
+  - keyboard
+  - grammar
+upforgrabs:
+  name: good first issue
+  link: https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

Comment:
@andrewtavis Can you please change this to https://github.com/scribe-org/Scribe-iOS/labels/good%20first%20issue